experimental/Schemes/src/IdealSheaves.jl

```diff
@@ -842,6 +842,7 @@ function maximal_associated_points(I::AbsIdealSheaf;
     covering=default_covering(scheme(I)),
     algorithm::Symbol=:GTZ
   )
+  return _maximal_associated_points(I; covering, algorithm, use_decomposition_info=true)
```

Comment:
The "old" routine was not necessarily returning false results, but it was just hung up on the particular input from our book chapter and never finished. I found that with `_maximal_associated_points` we get through. Since there is no general speed regression from this (as I already measured on another account), I suggest to just switch to the new implementation.

Reply:
If disabled like this, the old code will just rot. Either it is to be removed completely (but then we should first investigate why it hangs and be sure that the new implementation is reliably faster and not just on this notebook),  or we keep it alive, then both implementations have to be tested.

Reply:
@afkafkafk13 : The code in question to be removed is your "old" code for decomposing ideal sheaves. It stems from a time where a lot of the lazyfication had not yet been implemented. Now it can in principle be rewritten in a much more conceptual form with no significant deficits in speed. But I think, you are still the expert on this, since you have probably implemented this already several times in previous versions even in Singular. 

Hence my question: Do you think that in the new implementation in `_maximal_associated_points` I have overlooked some tuning that you have thought of but I didn't?

Reply:
The new code looks really fine. Much better than the one for last year. The whole environment has made a lot of progress. 

Please add a test for the roughly the following (before removing the old code):
Create a 2-dim. scheme X with two disjoint parts, choose an ideal sheaf on it which has at least 3 components describing subschemes of dimensions 1 and 0 respectively (so not pure-dimensional). Then blow up in a point on one component of X and then in a point on the other. Now decompose the  strict transform of the ideal sheaf.

This request is not about an optimization, but just to have a simple and inexpensive worst-case scenario in the tests.

Reply:
Thanks for having a look. But does such a test not already exist for the old code?

Reply:
No it doesn't, but I wanted to add something of that kind the next time I touch code in IdealSheaves.jl. Between the time of its writing and now, we got blow-up, normalization, lazy gluings and some more stuff. The code here needs to be tested with these in some inexpensive way.... If I come up with a good example, I will let you know.

Reply:
I wrote a test now. But I'm waiting with pushing it to see the current test run finish first.